### PR TITLE
Leaderboard standalone script

### PR DESCRIPTION
**Pull request by HxxxxxS**
Didn't get these two squashed since i had already pushed the first commit to github. See #144 for what this PR is.

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 // @version 4.3
 // @match *://steamcommunity.com/minigame/towerattack*
 // @match *://steamcommunity.com//minigame/towerattack*
+// @grant GM_xmlhttpRequest
```
Comment (xPaw) [inline autoPlay.user.js:8]: why?

Comment (HxxxxxS) [inline autoPlay.user.js:8]: shit, that was in the initial commit, didn't remove it again

man. 15. jun. 2015 kl. 19.53 skrev Pavel notifications@github.com:

**Comment by iakat** [comment]
My API now returns a new value, `position` - mind removing the position calculation and use that instead?

**Comment by HxxxxxS** [comment]
sure. how often do you want me to check by the way? thought i'd run it by
you if you want it to be like 10 mins or something.

man. 15. jun. 2015 kl. 20.00 skrev Simone notifications@github.com:

**Comment by iakat** [comment]
It should be fine as it is.

**Pull request closed without merging**